src/pages/ReceiverDetailsEdit.tsx

```diff
@@ -19,12 +19,9 @@ import { InfoTooltip } from "components/InfoTooltip";
 import { LoadingContent } from "components/LoadingContent";
 import { ErrorWithExtras } from "components/ErrorWithExtras";
 
-import {
-  ReceiverDetails,
-  ReceiverEditFields,
-  ReceiverVerification,
-} from "types";
+import { ReceiverDetails, ReceiverEditFields } from "types";
 import { useUpdateReceiverDetails } from "apiQueries/useUpdateReceiverDetails";
+import { NotificationWithButtons } from "components/NotificationWithButtons";
```

Comment:
Nit: We usually group imports as much as possible. This line should follow other "components" imports (line 21). And "types" is usually the last import (followed by relative imports if there are any).

Reply:
Thanks for pointing that out!

Reply:
https://github.com/stellar/stellar-disbursement-platform-frontend/pull/213/commits/2548eb8cabb1589b80902cdb28ee5f278bf6260f